eng/CredScanSuppression.json

```diff
@@ -47,7 +47,8 @@
                 "asdsadsad",
                 "Kg==",
                 "eyJhdWQiOiJodHRwczovL3N0b3JhZ2UuYXp1cmUuY29tIiwiaXNzIjoiaHR0cHM6Ly9zdHMud2luZG93cy1wcGUubmV0L2FiMWY3MDhkLTUwZjYtNDA0Yy1hMDA2LWQ3MWIyYWM3YTYwNi8iLCJpYXQiOjE1MTE4NTk2MDMsIm5iZiI6MTUxMTg1OTYwMywiZXhwIjo5OTk5OTk5OTk5LCJhbGciOiJIUzI1NiJ9",
-                "eyJhbGciOiJub25lIiwidHlwIjoiSldUIn0.eyJlbWFpbCI6IkJvYkBjb250b3NvLmNvbSIsImdpdmVuX25hbWUiOiJCb2IiLCJpc3MiOiJodHRwOi8vRGVmYXVsdC5Jc3N1ZXIuY29tIiwiYXVkIjoiaHR0cDovL0RlZmF1bHQuQXVkaWVuY2UuY29tIiwiaWF0IjoiMTYwNzk3ODY4MyIsIm5iZiI6IjE2MDc5Nzg2ODMiLCJleHAiOiIxNjA3OTc4OTgzIn0."
+                "eyJhbGciOiJub25lIiwidHlwIjoiSldUIn0.eyJlbWFpbCI6IkJvYkBjb250b3NvLmNvbSIsImdpdmVuX25hbWUiOiJCb2IiLCJpc3MiOiJodHRwOi8vRGVmYXVsdC5Jc3N1ZXIuY29tIiwiYXVkIjoiaHR0cDovL0RlZmF1bHQuQXVkaWVuY2UuY29tIiwiaWF0IjoiMTYwNzk3ODY4MyIsIm5iZiI6IjE2MDc5Nzg2ODMiLCJleHAiOiIxNjA3OTc4OTgzIn0.",
+                "clientSecret"
```

Comment:
Can you instead use one of the existing placeholders instead of adding another one? `__CLIENT_SECRET__` for example?

Reply:
The leaked secret is on this particular line: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/SessionRecords/NotebookClientLiveTests/TestGetNotebook.json#L264 and the beginning of the leaked secret is "_GLOBAL_CONFIG_DICT[\\\u0022clientSecret\\\u0022]\\n\u0022'
I'm not sure how I could use an existing placeholder for a part of the ResponseBody, is there a way I should modify the test? What if the test gets re-recorded?

Reply:
Are you sure that particular value is what is getting flagged? Do you have a report run before and after your change?

Reply:
It is indeed eliminate the errors.
Before:<img width="312" alt="image" src="https://user-images.githubusercontent.com/48036328/149224987-dfb018ed-55c1-49d5-8d7f-6781fc2d4a47.png">
After: <img width="604" alt="image" src="https://user-images.githubusercontent.com/48036328/149224961-79634a10-0b4f-4556-86ee-e91de66e744b.png">

Reply:
I checked "__CLIENT_SECRET__", it is only appear at this [file]("https://github.com/Azure/azure-sdk-for-net/blob/23f25364fbd97a8b3d3b82e8887e3e82b60597ac/sdk/resourcemanager/Azure.ResourceManager/docs/AuthUsingEnvironmentVariables.md)

"clientSecret" appears in responseBoby, but .net has no sanitizer for this to redact the keyword.

I suggested we keep both keyword in placeholders for simplicity. Or we reuse one of the keyword in [README]("https://github.com/Azure/azure-sdk-for-net/blob/23f25364fbd97a8b3d3b82e8887e3e82b60597ac/sdk/resourcemanager/Azure.ResourceManager/docs/AuthUsingEnvironmentVariables.md) to substitute the "__CLIENT_SECRET__" one in suppression file.